doc/.gitignore

```diff
@@ -1,2 +1 @@
-*.rst
```

Comment:
API `rst` files are currently generated automatically, so you want to keep them uncommited.

Reply:
but thats how you create documentation in sphinx. that is an absolute must that is removed...

Reply:
i can add api here to filter the generated api out

Reply:
Yes I would use `api/*.rst` instead. The files in that folder are generated by `sphinx-api`.

Reply:
I used `api/` for now